include/abb_librws/rws_client.h

```diff
@@ -574,6 +574,16 @@ class RWSClient : public POCOClient
    * \return RWSResult containing the result.
    */
   RWSResult endSubscription();
+
+  /**
+   * \brief Close the active subscription connection.
```

Comment:
```suggestion
   * \brief Force close an active subscription connection.
```
Slightly more accurate description.